src/components/SearchInput/index.js

```diff
@@ -85,17 +104,17 @@ class SearchInputContainer extends React.Component {
     return (
       <SearchInput
         onBack={this.props.onBack}
-        onChangeText={this.handleChangeText}
+        onChangeText={(value) => { this.handleChangeText(value); }}
```

Comment:
Use back `this.handleChangeText` since we're using arrow function above, `this` will be bounded